consai_examples/consai_examples/decisions/attacker.py

```diff
@@ -66,6 +68,21 @@ def inplay(self, robot_id):
                 self._act_id = ID_IN_THEIR_DEFENSE
             return
 
+        # 指定座標に向けてシュートする
+        if self._ball_state == FieldObserver.BALL_IS_IN_THEIR_SIDE:
+            if self._act_id != ID_SHOOT:
+                shoot_point = FieldObserver.get_shoot_point(robot_id)
+                if len(shoot_point) > 0:
+                    self._operator.shoot_to(robot_id, shoot_point[0], shoot_point[1])
+                self._act_id = ID_SHOOT
```

Comment:
インデントがずれていませんか？shoot_pointがなくても、かならずID_SHOOTが設定されます。
```suggestion
                if len(shoot_point) > 0:
                    self._operator.shoot_to(robot_id, shoot_point[0], shoot_point[1])
                    self._act_id = ID_SHOOT
```